sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 ### Breaking Changes
 
+- Removed from `Azure::Core::Http::Request`:
+  - `SetUploadChunkSize`.
```

Comment:
```suggestion
  - `SetUploadChunkSize()`.
```

Reply:
We use `()` to make it clear it's a method that is being removed rather than a type or field.

Reply:
CL feedback fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/2037